lib/src/anim_nav/animated_nav_tiles.dart

```diff
@@ -215,6 +215,7 @@ class AnimatedNavigationTiles extends StatelessWidget {
                   padding: items.badgePadding,
                   child: IconWidget(
                     item: items,
+                    color: items.selectedColor,
```

Comment:
Unnecessary `color` parameter: The color parameter seems redundant. The existing `item` property already provides the necessary color information.